docs/src/main/paradox/producer.md

```diff
@@ -91,12 +91,12 @@ Java
 
 ## Producing messages
 
-Sinks and flows accept implementations of @scaladoc[ProducerMessage.Envelope](akka.kafka.ProducerMessage$$Envelope) as input. They contain an extra field to pass through data, the so called `passThrough`. Its value is passed through the flow and becomes available in the @scaladoc[ProducerMessage.Results](akka.kafka.ProducerMessage$$Results)' `passThrough()`. It can for example hold a @scaldoc[ConsumerMessage.CommittableOffset](akka.kafka.ConsumerMessage$$CommittableOffset) or @scaldoc[ConsumerMessage.CommittableOffsetBatch](akka.kafka.ConsumerMessage$$CommittableOffsetBatch) (from a @apidoc[Consumer.committableSource](Consumer$)) that can be committed after publishing to Kafka. 
+Sinks and flows accept implementations of @apidoc[ProducerMessage.Envelope] as input. They contain an extra field to pass through data, the so called `passThrough`. Its value is passed through the flow and becomes available in the @apidoc[ProducerMessage.Results]' `passThrough()`. It can for example hold a @apidoc[akka.kafka.ConsumerMessage.CommittableOffset] or @apidoc[ConsumerMessage.CommittableOffsetBatch] from a @apidoc[Consumer.committableSource](Consumer$) that can be committed after publishing to Kafka. 
```

Comment:
Does `CommittableOffset` need to be fully qualified in this case?

```suggestion
Sinks and flows accept implementations of @apidoc[ProducerMessage.Envelope] as input. They contain an extra field to pass through data, the so called `passThrough`. Its value is passed through the flow and becomes available in the @apidoc[ProducerMessage.Results]' `passThrough()`. It can for example hold a @apidoc[ConsumerMessage.CommittableOffset] or @apidoc[ConsumerMessage.CommittableOffsetBatch] from a @apidoc[Consumer.committableSource](Consumer$) that can be committed after publishing to Kafka. 
```